notebooks/estimate_resource_usage.ipynb

```diff
@@ -0,0 +1,591 @@
p7q+rWuXP7TD117vM8d7SYWisqlvW3NdW5ysdqp9PLv/zVgyf7MdPi3K/e\n3fJIq0MPHpzGyNpT9m+SgYHa3/kcpl48b/Xvq90OPE0LzPamlErT0cKZ1KPujoijM/MigIh4IbVG\n3aRYqKmeXtmPsvRz78zMZPVNw5zztbXcdOsWHvyAWfzla/bi2csX7rjg43uP35eTzz6Opwyv5LDR\nnmexB6vmL+fqwVrPs6r3nttvnzk9sR9Q2r7Y3pIa+xS13pifBMjM6yLifKZQYG51+P/odSLqe09G\nXW/JWVE3T/04PTRjnN6UEbWejvVz2Y/2whz/c3bt7Vnf+/Or/9l8HtNlm1Zy6ucHefqhi3cUdncU\nfre1UAQeu27bZP/Va+bMhrmzg7lzdt7mzQm2ZGvD/7fkbDZvTTYMJ9u314bub9uexY3d1m3fXpte\nYTrMngWxbWsZF4ud8lzlY5VdzOykXtmXXvpN0q1j0s4czJ7dl6ZXRwtnUo96E/D5iDijWL6dWsNu\nUizUVE+v7EeJ+q79lplcc/NmPnPxfdz4iy08cK9ZvPNVe/GcwxcyZ/auvT6WP3Y+Z75vf7562WLO\nWnU46zbPYmjedo46bIgzj9xjxvxd9cp+QCn7YntLamxBMS1G/boplTlbHf4/PFK7TsTYuWBzZPc5\nZcvQyjymy4dX8rGbl7P6p7v2BBwt7I4t9M6dEwzOHWBowa6vGfvasc/NGe+1s4O5c2v38+YGc2YH\nswbG7/3YzvD/0/+ivR6zIyOtFaLrn9s6zut2fU/tb2Lrtl3f++/fXlvGxWKhw3OV91Ixs1f2pZd+\nk3TrmLRTYK7U2f2IWAGsWLp0aVkhSN02pcKZOaJ+kJk/Bw6PiEXF8vpW3ztRjlioqZ5e2Y+STKn9\nNtO+S354c63H8o9/vpl99pjFO165J8996qLdCsv19ttnDie+Yp9KD8NuRa/sB0z7vtjekhr7bUQ8\nguI6vBHxEuDXrb55vBxpZ/h/q9eJyKLgPHpxspGxy6MF6qI4XX/hsrHLOcEF0cZe+Oxd//yblueW\n/Y9/2G9HYXjO7O7METsV3exlOjAQzB2oFcy77RvfvWfaLxYLU5urfLwc6aViZi/tS6/8JunWMYls\n8dKPEfEd4PQxZ/ffOtmrYnbKsmXLcvXq1WWGII0rItZk5rIObKftwlk9c0RV1akcGWe7h2bmNa2+\n3hxRVXUiRzrVfqt6nvzop8Oc8/W1/Oh/NvOAJbN4zXMX87wjFk3Lj1mVp5PfI7a31KummicRcRBw\nNrXRMPcCvwBe0+5US/U5csYFd7H28ksbFjO/seB57PnMZ1f6pNmL3nELb/7dqQ2Lmb+NvThr77dx\n4ccPmsbI2nfH3Vs54f/ettt80qNuHXgY5+5R/QsvTuZvq0PtrXHnKs/MlqeSGe975I67t/LVy+7j\n0lXrdilmHjODipmjemlfekWrx6TVHGmnB3NHhyFLak39D512C2dSn3oz8Iayg5Aqoqfbbz/+Wa2w\n/MObN7PX4gH+/KV78oKnW1hW+2xvSePLzFuAoyJiITCQmeumus1eGTLfK3PLQu/0Mi3xb6vjc5WD\no5PUXZ0+Ji0XmKcyDFlSx1g4k5rITHNEKvRq++2GWzZzztfWsuYnw+y5eIATXrIHK56+iHlzB8oO\nTb3B9pY0RmZuGH081ZMwFjOrqReG/5f4t9XxucqlmaadHsyAZ/elMlk4k3YVEZeNHeo/3jqp3/VK\n++2mWzdz7tfWsurGYfZYNMCbjt2Do/9gEYMWltVBtrekpqZ8EsZiZjX1Qi/Tkv62pjRXudQL2i4w\nj+HZfalLLJxJE4uIQWABsHdE7AmMdhdYDOxXWmDSzDDj2m8/vW0L53ztPn5w/TCLFw5w/DF78MI/\nXMT8eRaWNTW2t6T2deokjMVMdUsJf1snUpur/OCIuINirvLp+nCpCqZUYPbsvtR5Fs6klrwReDuw\nL7CGnXlyP3DGRG+SNLPab//zqy2c+/W1XHndJoYWDPB/jl7CMc8YYsGghWVNje0tqTWehGmuFwrl\nmppuzFUuzTQtF5j9YpGmjYUzqYnMPBU4NSLekpmnlx2PVFUztf3289trheXv/WgTi+YHr1+xhGOf\nMcTC+RaW1TG2t6QGPAkjta+Tc5VLM03TAnNVv1giYgWwYunSpWWFIHVFpwpn5oj6QWaeHhFHAAdS\n952Wmec1e685ol7WqfbbdOfJL+6sFZav+OEmFs4P/vSPl/DiZw2xyMKyOsz2ltRUR07CmCPqYy1N\nSWaOqFdEZjZ+QcTb2PnFcge7frF8KjNLPcO/bNmyXL16dZkhSOOKiDWZuWyK25hU4ayeOaKq6lCO\nfBZ4BHAtsL1YnZn51la3YY6oqqaSI51uv3U7T2799VbO+/pavvvDjcyfF7z4mUO85MjFDC2wsKyJ\ndeJ7pNiO7S31rKnmSadGi5kjqqpOfZdMlTmiqmo1R5r2YHYYslSOiQpnQFs/eKQetww4JJudLZX6\nzExpv932v1s57xtr+a81GxmcG7zqjxbz0qOGWLxwVtmhqU/Y3pIam8poMalfzNQpyaROankOZr9Y\npGln4Uxq7nrgwcCvyw5EqqKqtt9uv6tWWP7Pqzcyd27w8mcv5uVHDbFkkYVlTTvbW1IDnoSRJlbV\nKWWlMrRzkT+/WKTpZeFMam5v4MaIWAVsHl2ZmUeXF5JUHVVrv91x91Y+d8n9fGfVBubMCl565BAv\nf/Zi9hiysKzS2N6SGvMkjDQxLxgrFVouMOMXizTdLJxJzZ1UdgBSxVWi/fbr327jc5es5VsrNzB7\nVm2O5Zc/ezF7LbawrNLZ3tKMc8fdW7nw0nu5bNV67t88i8XztnPkYYt40VF7st8+czr9cZ6EkSYw\nU6Ykk6ZDOwVmv1jU96a5MXdSpzco9ZrM/G7ZMUgV15X2W6vfh//7u2
```

Comment:
Would it make sense to add a `save` flag here instead of commenting out the line?

Reply:
I should probably just not comment it!